src/main/kotlin/com/rootsid/wal/library/mongoimpl/document/BlockchainTxLogDocument.kt

```diff
@@ -0,0 +1,20 @@
+package com.rootsid.wal.library.mongoimpl.document
+
+import com.rootsid.wal.library.wallet.model.BlockchainTxAction
+import com.rootsid.wal.library.wallet.model.BlockchainTxLog
+import io.iohk.atala.prism.api.models.AtalaOperationStatus
+import io.iohk.atala.prism.api.models.AtalaOperationStatusEnum
+import kotlinx.serialization.Serializable
+
+@Serializable
```

Comment:
Remove this and make the BlockchainTxLog implement Serializable

Reply:
ok